feather/regression_test.py

```
@@ -0,0 +1,968 @@
+            # Read until connection closes or timeout
+            # For HTTP responses, we need to read the full response
+            chunks = []
+            try:
+                while True:
+                    chunk = await asyncio.wait_for(reader.read(1024), timeout=2.0)
+                    if not chunk:
+                        break
+                    chunks.append(chunk)
+            except asyncio.TimeoutError:
+                # Timeout means no more data is arriving, exit the loop
+                break
```

Ruff (0.14.0)

`break` outside loop

(F701)